src/plugins/discover/public/application/main/components/layout/discover_documents.tsx

```diff
@@ -416,12 +408,12 @@ function DiscoverDocumentsComponent({
                   rowHeightState={rowHeight}
                   onUpdateRowHeight={onUpdateRowHeight}
                   isSortEnabled={true}
-                  isPlainRecord={isTextBasedQuery}
+                  isPlainRecord={isEsqlMode}
```

Comment:
Why here we are keeping the `isPlainRecord` and we dont change the prop to `isEsqlMode `? I see that we are doing this in other places